en/docs/install-and-setup/setup/security/user-account-management-for-dev-portal.md

```diff
@@ -0,0 +1,178 @@
+# User Account Management for the Developer Portal
+
+The following sections explain how to manage Developer Portal related user accounts.
+
+## Password recovery
```

Comment:
is this applicable only for dev portal?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3